src/DotLiquid/NamingConventions/RubyNamingConvention.cs

```diff
@@ -25,6 +25,11 @@ public StringComparer StringComparer
             get { return StringComparer.OrdinalIgnoreCase; }
         }
 
+        public StringComparer ConditionComparer
+        {
+            get { return StringComparer.OrdinalIgnoreCase; }

```

Comment:
Or maybe you are saying that you want to throw an error if the condition contains any non-lowercase characters? That would be in line with your comment on the CSharp ConditionComparer as well. I'd have to assume that non-alphanumeric characters would be allowed (at least the same characters that the regex allows: [=!<>a-zA-Z_]). I wouldn't want to break backwards compatibility with anyone who has implemented custom operators: https://github.com/dotliquid/dotliquid/blob/master/src/DotLiquid.Tests/ConditionTests.cs#L171